EIPS/eip-7069.md

```diff
@@ -114,10 +115,13 @@ Before the 63/64th rule was introduced, it was required to calculate available g
 
 We have changed the ruleset:
 
-1. Removed the call depth check.
-2. Use the 63/64th rule, but
-    2a. ensure that at least 5000 gas is retained prior to executing the callee,
-    2b. ensure that at least 2300 gas is available to the callee.
+1. Use the 63/64th rule, but
+    - Ensure that at least `MIN_RETAINED_GAS` gas is retained prior to executing the callee,
+    - Ensure that at least `MIN_CALLEE_GAS` gas is available to the callee.
+
+Removing the call stack depth check was initially considered, but this would be incompatible with the original `*CALL` instructions, as well as `CREATE*` instructions, which can be intertwined with the new `EXT*CALL` instructions in the call stack. As such, keepeing the call stack depth involves the least change for legacy code.
```

Comment:
maybe
```suggestion
Removing the call stack depth check was initially considered, but this would be incompatible with the original `*CALL` instructions, as well as `CREATE*` instructions, which can be intertwined with the new `EXT*CALL` instructions in the call stack. As such, keeping the call stack depth involves no change affecting legacy code.
```